### ISSUE TITLE
Suggestion 

### ISSUE THREAD

**Issue by AlexGagne10** [FloppyDiskUser/WeezBot#4]
I have a few suggestions who could interest you. 
- Manual Snipe by coordinate [ planned for next update ]
- Add Pokemon to gym. –> If the place is empty
- In inventory: Add pokeball, incense count, etc. [ Added ]
- Choose favourite Pokemon by click right
- Choose Pokemon buddy
- Pokemon buddy km walked
- Add possibility to add  break interval and length
- Google Maps Api [ never #MicrosoftFanBoy ]
- Add logs: Pokemon caught, Eggs Hatched
- Choose Team [ currently working to get this working, will be within the next updates ]
- Maybe for security add something to use adb to check device information.
- Favourite Pokemon can’t be transfers by mistake.
- Possiblity to farm only ultra ball or great ball 
  Thank you again for your bot. Good work

**Comment by arashranjbar** [comment]
"Choose Team"
i like that idea

**Comment by FloppyDiskUser** [comment]
@arashranjbar hey, I like that idea too but its really time intensive, and I didn't earned anything with this bot and I have a lot of other projects to do, and you'll probably understand that I'll do these first.

Edit: this shouldn't mean I wont do it. its just an explaining why it could take some time

**Comment by arashranjbar** [comment]
make topics in forums and make the bot famous :D

**Comment by FloppyDiskUser** [comment]
@arashranjbar yeah, but I am not very good at this or know where I have to post it and dont want to spend money on advertisement :D I thought about youtube and wrote some messages to youtubers if they can take a look. Until now no one replied :(
If you know a better way I would love to hear that idea

**Comment by arashranjbar** [comment]
1. make a video of how the bot works (show github like in video)
2.upload it on youtube (write github like on youtube  Description)
3.make a topic on mobile & pokemon forums
4.post youtube & github link in those topics with some text and bot's screenshots
thats it. completely free

**Comment by FloppyDiskUser** [comment]
@arashranjbar thanks for these tipps, I try as many as I can :)

**Comment by arashranjbar** [comment]
some of pokemon aren't in their actual size So gif does not display them in good quality
list: ekans/paras/nidoranFemale/Bellsrpout/poliwag and ...

please fix them
thank you

![2016-09-16 00_44_59-weezbot pokemon go bot](https://cloud.githubusercontent.com/assets/19646232/18565920/fadfbeac-7ba6-11e6-9e2b-d7494d9b46bf.png)

**Comment by FloppyDiskUser** [comment]
@arashranjbar hey, already fixed will be with the next update.

**Comment by FloppyDiskUser** [comment]
@AlexGagne10 what do you exactly mean with

> Possiblity to farm only ultra ball or great ball

**Comment by arashranjbar** [comment]
thank you very much
another thing that is bother me it's that we don't have back button here:
![2016-09-16 01_42_43-weezbot pokemon go bot](https://cloud.githubusercontent.com/assets/19646232/18567804/11c54f12-7baf-11e6-8c72-b5295c2d5960.png)

**Comment by FloppyDiskUser** [comment]
@arashranjbar okay, tomorrow I'll do this and some other stuff.

**Comment by AlexGagne10** [comment]
I have an idea to promote your bot. 
—Make a Facebook page and ask people to share with friends. After example 100 shares you giveaway 2 account LVL 20, for example. 
—Write in github wiki how to use it. I can write in French for you if you want but English I really can’t lol.
—Like @arashranjbar say a video would be great too.
—Add donation by bitcoin for your security because with PayPal, Niantic could find you easily. 
—Discordapp is very great. Maybe you could create Weezbot discord.
—Every donation received 2 accounts 

PS: I have some account LVL 20 if you want to start with that!! And if you like the idea i will help you to make some great account.

And finaly for now lol. On GitHub your project missing CS2PHPCryptography.dll (package missing, and I can't find on nugget.com) same with POGO.NecroBot.Logic.dll (Package Nugget).

**Comment by FloppyDiskUser** [comment]
@AlexGagne10 hey, thank you for this long helpfull list , this things with the donation rewards and the github wiki are coming soon. 
Hey these Projects don't belong to me. I'll post links soon

**Comment by arashranjbar** [comment]
why i can't use bot? :((

**Comment by FloppyDiskUser** [comment]
@arashranjbar hey, sorry I hope I fixed it. If not please tell me
